### PR TITLE
In Due Time, time management tips and tricks

### PR DESCRIPTION
**Pull request by LouisPryer**
### Blog title
In Due Time
### Blog description
Blog post on some neat time management tips and tricks, along with some links to other time management sits and other to-do list blogs hosted on hungry geek
#### Author
- [X] I have checked for grammatical errors
- [X] I have checked for spelling errors
- [X] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [ ] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by pmcnr-hx** [comment]
I would suggest you wrap the paragraph lines.

File: _posts/2015-07-22-in-due-time.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+categories: Time Management
+author: louis_pryer
+tags: toggl pomodoro management
+comments: true
+excerpt: "Like many people, I find it a struggle to fit everything I need to do into a single day. With the deadline of my report edging ever closer, I have been experimenting with different time management tools, to help make the most out of the time given to me."
+---
+
+#In Due Time
+
+Like many people, I find it a struggle to fit everything I need to do into a single day. With the deadline of my report edging ever closer, I have been experimenting with different time management tools, to help make the most out of the time given to me. I have found a few neat tricks and tools that have worked for well for me, so I thought I would take the time to share these along with some other tips from other members of the HX team. If you can spare a few minutes, hopefully this might help you become more efficient in your work day!
```
Comment (pmcnr-hx) [inline _posts/2015-07-22-in-due-time.md]: "worked for well for me" → "worker well for me"

File: _posts/2015-07-22-in-due-time.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Starting off strong is the number one tool that has helped me immensely with my own time management, [Toggl](https://www.toggl.com/). This was recommended to me by our Trading Scrum Master, Chrissy Garnett, which I noticed she used during sprint estimation meetings to gauge how long they lasted, along with other meetings. Toggl is a simple way to create a timesheet of your activity, either during work hours or outside of it. With integrations to applications and services like GitHub and JIRA, along with project grouping and automatic detailed graphs of your tracked time, this has been an immense help with eliminating bottlenecks in my daily routine while allowing me to use my time as efficiently as possible.
```
Comment (pmcnr-hx) [inline _posts/2015-07-22-in-due-time.md]: "which I noted she used" → "which I noted using", maybe? not sure.

File: _posts/2015-07-22-in-due-time.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+![](http://grab.by/II8y)
```
Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: Image isn't working for me

File: _posts/2015-07-22-in-due-time.md (hunk 1 above)
Comment (hxpaul) [inline _posts/2015-07-22-in-due-time.md]: not a blocker just imo, "hopefully this might" sounds super mild, maybe just "this might" or "hopefully this will"

File: _posts/2015-07-22-in-due-time.md (hunk 3 above)
Comment (pmcnr-hx) [inline _posts/2015-07-22-in-due-time.md]: Should upload the screenshot as an attachment to this PR and link from there.

File: _posts/2015-07-22-in-due-time.md (hunk 2 above)
Comment (hxpaul) [inline _posts/2015-07-22-in-due-time.md]: not a blocker but I don't understand "starting off strong is" I thought it was the name of the tool.

Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: :+1:

File: _posts/2015-07-22-in-due-time.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+*Screengrab of my toggl dashboard, by labelling each time entry with a different project I can automatically generate pie charts of my day and how much time I have spent doing each project*
+
+
+Another popular technique used by some developers in the office is the [pomodoro technique](http://pomodorotechnique.com/). This technique involves working in intervals of a set period, most commonly 25 minutes, followed by a short break of 3-5 minutes with a longer break of up to 30 minutes after every fourth repetition. This technique is great to eliminate burnout like you would get in long, intense coding sessions while allowing you to time box tasks during each interval. One trap you need to avoid is to do work related tasks during your breaks. These breaks are to let your brain rest and reset for the next repetition. Grab a coffee and take a breather; you will thank me later.
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/34/Il_pomodoro.jpg/480px-Il_pomodoro.jpg)
+
+*Typically the timer is represented as a pomodoro tomato, due to the creator(Francesco Cirillo) using a similar tomato looking timer while studying at university*
```
Comment (hxpaul) [inline _posts/2015-07-22-in-due-time.md]: `creator` is not a function, missing space there I think

File: _posts/2015-07-22-in-due-time.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+![](http://grab.by/II8y)
+*Screengrab of my toggl dashboard, by labelling each time entry with a different project I can automatically generate pie charts of my day and how much time I have spent doing each project*
+
+
+Another popular technique used by some developers in the office is the [pomodoro technique](http://pomodorotechnique.com/). This technique involves working in intervals of a set period, most commonly 25 minutes, followed by a short break of 3-5 minutes with a longer break of up to 30 minutes after every fourth repetition. This technique is great to eliminate burnout like you would get in long, intense coding sessions while allowing you to time box tasks during each interval. One trap you need to avoid is to do work related tasks during your breaks. These breaks are to let your brain rest and reset for the next repetition. Grab a coffee and take a breather; you will thank me later.
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/34/Il_pomodoro.jpg/480px-Il_pomodoro.jpg)
```
Comment (pmcnr-hx) [inline _posts/2015-07-22-in-due-time.md]: Fill `[]` description.

File: _posts/2015-07-22-in-due-time.md (hunk 2 above)
Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: The last sentence here is really long, I would consider breaking it up into a few shorter sentences like

"Toggle comes with integrations to applications and services like GitHub and JIRA. It's also go project grouping and automatic detailed graphs of your tracked time.
Overall I would say Toggl has been an immensely helpful tool with eliminating bottlenecks in my daily routine while allowing me to use my time as efficiently as possible."

I don't think exactly that, but I find shorter sentences easier to read.

File: _posts/2015-07-22-in-due-time.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Starting off strong is the number one tool that has helped me immensely with my own time management, [Toggl](https://www.toggl.com/). This was recommended to me by our Trading Scrum Master, Chrissy Garnett, which I noticed she used during sprint estimation meetings to gauge how long they lasted, along with other meetings. Toggl is a simple way to create a timesheet of your activity, either during work hours or outside of it. With integrations to applications and services like GitHub and JIRA, along with project grouping and automatic detailed graphs of your tracked time, this has been an immense help with eliminating bottlenecks in my daily routine while allowing me to use my time as efficiently as possible.
+
+![](http://grab.by/II8y)
+*Screengrab of my toggl dashboard, by labelling each time entry with a different project I can automatically generate pie charts of my day and how much time I have spent doing each project*
```
Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: project.

File: _posts/2015-07-22-in-due-time.md (hunk 4 above)
Comment (pmcnr-hx) [inline _posts/2015-07-22-in-due-time.md]: _pomoro_ is _tomato_ in Italian, so _pomodoro tomato_ seems redundant. Maybe rephrase?

Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: space after creator

Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: university.

**Comment by hxpaul** [comment]
Nice work, thought it feels like there should be one more alternative technique / method / system in there, three is the magic number. Not a blocker though. 
I like how you end on a question there too.

File: _posts/2015-07-22-in-due-time.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
++ If possible, schedule out time for interruptions and distractions. 
++ Remove as many distractions as possible while working on important or urgent tasks. Disable things like email notifications until the task is completed or near completion.
++ Expect the unexpected as your plan for the day may change on a whim. Don't let this affect important tasks that need doing. Remove unimportant tasks from your todo list to compensate for change.
++ For more To-do list tricks and tips, our own [@JackCrawford](https://twitter.com/jackdcrawford) and [@RobHuzzey](https://twitter.com/theHuzz) have shared their own processes of organising tasks. Why not give [Making Stuff Happen](http://hungrygeek.holidayextras.co.uk/workflow/2015/02/24/making-stuff-happen/) and [To-dos that stick](http://hungrygeek.holidayextras.co.uk/workflow/2015/02/20/todos-that-stick/) a quick read.
+
+If you have any other neat tricks to help balance your time, let us know in the comments below. I am always looking to experiment with new techniques for managing my own productivity.
+ 
+
```
Comment (lukefrake) [inline _posts/2015-07-22-in-due-time.md]: Don't need these extra lines

**Comment by lukefrake** [comment]
Really nice post!

**Comment by pmcnr-hx** [comment]
:+1:

**Comment by lukefrake** [comment]
:+1:
